buildworker.rst

```diff
@@ -80,42 +80,42 @@ For Mac:
     * Create a buildbot user using the OS/X control panel user admin.  It
       should be a "standard" user.
     * Log in as the buildbot user.
-    * Install either the Python 2.7 bundle from python.org [#]_, or pip.
+    * Install either the bundle from python.org [#]_, or pip.
     * Open a terminal window.
-    * Execute ``pip install buildbot-slave``.
+    * If using pip; execute ``pip install buildbot-worker``.
```

Comment:
I would structure this as:

- Create a buildbot user ...
- Log in as the buildbot user
- Install buildbot, either from python.org or by running pip install buildbot-worker

Reply:
Thanks, this is much clearer.